tools/DfmHttpService/Program.cs

```diff
@@ -19,7 +19,7 @@ public static void Main(string[] args)
                     new ExitHandler()
                 });
 
-            var service = new DfmHttpServer(handler, args.Length > 0 ? args[0] : null);
+            var service = new DfmHttpServer(handler, "localhost", args.Length > 0 ? args[0] : null);
```

Comment:
As I mentioned, DfmHttpServer/Program.cs does not do arg "parsing" per se. It passes arg[0] directly into the function DfmHttpServer(). The better thing would be to parse the args like docfx/Program.cs does. That was a bit of machinery I didn't feel comfortable doing, e.g., -n '*' 5000. Or, -n '*' -p 5000?